PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-arm/src/main/resources/META-INF/azure-intellij-plugin-arm.xml

```diff
@@ -0,0 +1,18 @@
+<idea-plugin>
+    <extensions defaultExtensionNs="com.microsoft.tooling.msservices.intellij.azure">
+        <azureExplorerContributor implementation="com.microsoft.azure.toolkit.ide.arm.ArmExplorerContributor"/>
+        <actions implementation="com.microsoft.azure.toolkit.ide.arm.ArmActionsContributor"/>
+        <actions implementation="com.microsoft.azure.toolkit.intellij.arm.IntellijArmActionsContributor"/>
+    </extensions>
+    <extensions defaultExtensionNs="com.intellij">
+        <fileEditorProvider implementation="com.microsoft.azure.toolkit.intellij.arm.property.DeploymentPropertyViewProvider"/>
+        <fileEditorProvider implementation="com.microsoft.azure.toolkit.intellij.arm.template.ResourceTemplateViewProvider"/>
+        <fileTypeFactory implementation="com.microsoft.azure.toolkit.intellij.arm.language.ARMTemplateFileTypeFactory"/>
+        <lang.syntaxHighlighterFactory language="arm"
```

Comment:
We may investigate whether we could remove the arm language and use json directly